06_deploy_bootstrap_vm.sh

```diff
@@ -24,6 +25,9 @@ pullSecret: |
 sshKey: |
   ${SSH_PUB_KEY}
 EOF
+    set -x
+    echo "ocp/install-config.yaml (without pull secret)"
+    sed -e 's/.*auth.*/***PULL_SECRET***/g' ocp/install-config.yaml
```

Comment:
You do this twice, so how about a safe_print_ocp_install_config() function?

Reply:
Oh, right - the other one isn't install-config.yaml

Reply:
Would it be safer to use yq to edit the yaml vs sedding it?

Reply:
Done, I'm not sure how to replace the other sed with yq, the path isn't so simple, It would be something like 
yq '.spec.storage[where path: /var/lib/kubelet/config.json].contents = "***PULL_SECRET***"'

I don't know if the part in the square brackets is possible, any ideas?

Reply:
Hmm not sure either, perhaps we'd merge another map that overwrites the current pull secret?

Tbh I hadn't spotted this second more complex path so maybe the sed is fine in that case

Reply:
I'm happy to go with sed, a complex yq might be more brittle anyways.